<ui version="4.0" >
 <class>Gui</class>
 <widget class="QMainWindow" name="Gui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SimMLST</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Number of isolates</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3" >
     <widget class="QLineEdit" name="isoEd" >
      <property name="whatsThis" >
       <string/>
      </property>
      <property name="inputMask" >
       <string/>
      </property>
      <property name="text" >
       <string>100</string>
      </property>
      <property name="frame" >
       <bool>true</bool>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Lengths of gene fragments</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3" >
     <widget class="QLineEdit" name="fragsEd" >
      <property name="text" >
       <string>400,400,400,400,400,400,400</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Mutation rate (theta)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3" >
     <widget class="QLineEdit" name="thetaEd" >
      <property name="text" >
       <string>100</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="label_4" >
      <property name="text" >
       <string>Recombination rate (rho)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3" >
     <widget class="QLineEdit" name="rhoEd" >
      <property name="text" >
       <string>100</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0" >
     <widget class="QLabel" name="label_5" >
      <property name="text" >
       <string>Mean tract length (delta)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="3" >
     <widget class="QLineEdit" name="deltaEd" >
      <property name="text" >
       <string>500</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0" >
     <widget class="QLabel" name="label_10" >
      <property name="text" >
       <string>Population size model</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" >
     <widget class="QLineEdit" name="popSizeEd" >
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2" >
     <widget class="QToolButton" name="popSizeShow" >
      <property name="text" >
       <string>Show</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="4" >
     <widget class="Line" name="line" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0" >
     <widget class="QLabel" name="label_6" >
      <property name="text" >
       <string>Save data to...</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="2" >
     <widget class="QLineEdit" name="dataTo" />
    </item>
    <item row="7" column="3" >
     <widget class="QToolButton" name="dataSelect" >
      <property name="text" >
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" >
     <widget class="QLabel" name="label_9" >
      <property name="text" >
       <string>Save local trees to...</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="2" >
     <widget class="QLineEdit" name="ltTo" />
    </item>
    <item row="8" column="3" >
     <widget class="QToolButton" name="ltSelect" >
      <property name="text" >
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" >
     <widget class="QLabel" name="label_7" >
      <property name="text" >
       <string>Save clonal genealogy to...</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1" colspan="2" >
     <widget class="QLineEdit" name="cgTo" />
    </item>
    <item row="9" column="3" >
     <widget class="QToolButton" name="cgSelect" >
      <property name="text" >
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" >
     <widget class="QLabel" name="label_8" >
      <property name="text" >
       <string>Save DOT graph to...</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="2" >
     <widget class="QLineEdit" name="dotTo" />
    </item>
    <item row="10" column="3" >
     <widget class="QToolButton" name="dotSelect" >
      <property name="text" >
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="3" >
     <widget class="QCheckBox" name="amEd" >
      <property name="text" >
       <string>Save ancestral material in the DOT graph</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="4" >
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="help" >
        <property name="text" >
         <string>Help</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="run" >
        <property name="text" >
         <string>Simulate!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
